src/install-opentofu.ps1

```diff
@@ -23,42 +23,68 @@ Path to cosign. (Standalone installation only.)
 OIDC issuer for cosign signatures. (Standalone installation only.)
 .PARAMETER cosignIdentity
 Identity for the cosign signature. (Standalone installation only.)
+.PARAMETER gpgPath
+Path for the GPG installation. (Standalone installation only.)
+.PARAMETER gpgURL
+URL to download the GPG key from. (Standalone installation only.)
+.PARAMETER gpgKeyID
+GPG key ID / fingerprint to expect. (Standalone installation only.)
 .PARAMETER skipVerify
-Skip cosign integrity verification. (Standalone installation only; not recommended.)
-.Parameter skipChangePath
+Skip cosign/GPG integrity verification. (Standalone installation only; not recommended.)
+.PARAMETER skipChangePath
```

Comment:
This comment about `skipChangePath` is incorrect:
```suggestion
Skip changing the user/system path to include the OpenTofu path
.PARAMETER skipChangePath
```

See https://github.com/opentofu/get.opentofu.org/blob/ddfdb1e4388db6f29285cace3d7e9cb77de188c7/src/install-opentofu.ps1#L238-L259
and https://github.com/opentofu/get.opentofu.org/blob/ddfdb1e4388db6f29285cace3d7e9cb77de188c7/src/install-opentofu.ps1#L565C49-L565C112

Reply:
My bad, comments in this script are *after* the parameter definition :upside_down_face:

Reply:
@janosdebugs — you might consider putting a newline space between each param to reduce this confusion in the future

Reply:
We rolled back the Powershell changes as they have issues, they will follow in a later PR. Thank you everyone!